packages/react-jss/src/types.js

```diff
@@ -9,7 +9,7 @@ export type Managers = {[key: number]: SheetsManager}
 
 type StyleSheetOptions = {
   ...StyleSheetFactoryOptions,
-  classNamePrefix: string
+  classNamePrefix: string | any
```

Comment:
```suggestion
  classNamePrefix: string | false
```

Reply:
It throws flow error `Cannot add options.context.sheetOptions.classNamePrefix and prefix because boolean literal false [1] is incompatible
with string`

Reply:
sure, you have to enable this in the type of sheet options

Reply:
I did, but I still get the same error. Should I commit without checks?

Reply:
yes if you want us to look at the problem